docs/yarn.lock

```diff
@@ -2,7 +2,7 @@
 # yarn lockfile v1
 
 
-swagger-ui-dist@^3.20.3:
-  version "3.20.3"
-  resolved "https://registry.yarnpkg.com/swagger-ui-dist/-/swagger-ui-dist-3.20.3.tgz#e099a02572c360abe1c8892beb85ff685f1c8e6b"
-  integrity sha512-2O+uWBROdj7WLIIWabJe8TlQJiGgr96rmayPzp+pek/iIcUYoqKG+0icJL09Z9bZAAk3nWOTjTwU53XmjtCN7w==
+swagger-ui-dist@^4.1.0:
```

Comment:
Can you note what the breaking changes were and how we are not affected?

Reply:
The breaking changes for v4 are regarding React, which does not affect us.

> For anyone not integrating SwaggerUI or SwaggerEditor as a React library, there are no breaking changes. The v4  releases will continue to support OpenAPI 3.0.x and Swagger 2.0 as before. For library maintainers, the only breaking change introduced is dropping support for older versions of React and Redux.

https://swagger.io/blog/news/what-s-new-in-swaggerui-v4-and-swaggereditor-v4/